packages/agent/src/actor.ts

```diff
@@ -583,9 +590,9 @@ function _createActorMethod(
             );
           }
         }
-      } else if (response.body && 'reject_message' in response.body) {
+      } else if (isV2ResponseBody(response.body) && response.body.reject_code !== undefined) {
```

Comment:
The 2nd check is already handled by `isV2ResponseBody`

```suggestion
      } else if (isV2ResponseBody(response.body)) {
```